modules/proc-switching-bf2-nic.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * networking/switching-bf2-nic-dpu.adoc
+
+:_content-type: PROCEDURE
+[id="proc-switching-bf2-nic_{context}"]
+= Switching Bluefield-2 from DPU mode to NIC mode
+
+Use the following procedure to switch Bluefield-2 from data processing units (DPU) mode to network interface controller (NIC) mode.  
+
+.Prerequisites
+
+* 
+
+.Procedure
+
+. Add the following labels to your `worker` nodes by entering the following commands:
++
+[source,terminal]
+----
+$ oc label node worker-05 node-role.kubernetes.io/sriov=
```

Comment:
These are just examples; The user would need to apply this to the relevant nodes, which would be 1 or more that they might have.